base-api.Tests/V1/Infrastructure/DatabaseEntityTests.cs

```diff
@@ -0,0 +1,10 @@
+using NUnit.Framework;
+
+namespace UnitTests.V1.Infrastructure
+{
+    [TestFixture]
+    public class DatabaseEntityTests
+    {
```

Comment:
I'm not sure we need to test this. If I'm understanding correctly`DatabaseEntity` is just declaring the table structure so should be covered by the gateway tests.

Reply:
Happy to remove 👍